lbz/exceptions.py

```diff
@@ -9,17 +12,24 @@
 
 
 class LambdaFWException(Exception):
+    """
+    Standarised for AWS Lambda exception class.
+    """
+
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
 
-    def __init__(self, message: str = ""):
+    def __init__(self, message: str = ""):  # pylint: disable=super-init-not-called
```

Comment:
And the last one but it is with us starting from the beginning of this journey: [#670993489](https://github.com/pdyba/lambdalizator/pull/35/files#r670993489).

You should use super init instead of disabling that, even if temporarily without passing any argument:
```python
def __init__(self, message: str = ""):
    super().__init__()
```